static/app/components/button.tsx

```diff
@@ -149,8 +167,14 @@ function BaseButton({
   disabled = false,
   tooltipProps,
   onClick,
+  eventName,
+  eventKey,
+  analyticsParams,
   ...buttonProps
 }: ButtonProps) {
+  const organization = useOrganization();
+  const routes = useRoutes();
```

Comment:
This is probably going to break a lot of tests.

Reply:
Also button is not always guaranteed to be used in the organization context.

Reply:
We only need this tracking within the org context, if the button is used outside the org context we don't need to track

Reply:
We don't have it yet, but Alberto has mentioned wanting a `useMaybeOrganization` that returns `Organization | null`, that's what we'd need to do this